Excluding tools/launcher test on OpenJ9

psoujany opened the pull request:
This PR fixes runtimes/openj9-openjdk-jdk11-zos/issues/652.

Signed-off-by: Pasam Soujanya psoujany@in.ibm.com

llxia reviewed (approved):
LGTM

llxia commented:
FYI @JasonFengJ9

JasonFengJ9 commented:
This test passes in other platforms, can it be limited on JDK11 `zOS` only?

smlambert reviewed (changes requested):
Please update to only exclude on zos platform (rather than generic_all)

psoujany commented:
Updated exclude list as per https://github.com/adoptium/aqa-tests/pull/2648#issuecomment-861835475. Removed this exclusion from other releases and limited only to JDK11 z/OS. Thank you.

JasonFengJ9 reviewed (approved):
LGTM

psoujany commented:
@Mesbah-Alam Could you please review the PR. Thanks.
